src/Profilers/MonitorProfiler/Communication/Messages.h

```diff
@@ -5,15 +5,26 @@
 
 #include <vector>
 
-enum class IpcCommand : short
+enum class ProfilerCommand : unsigned short
 {
-    Unknown,
-    Status,
     Callstack
 };
 
+enum class DotnetMonitorCommand : unsigned short
+{
+    Status
+};
+
+enum class CommandSet : unsigned short
+{
+    DotnetMonitor,
+    Profiler,
+    ManagedInProc
```

Comment:
I believe that this isn't reference anywhere in the native code but I understand that this is probably a placeholder so that the value isn't accidentally taken by a new command set. I would suggest adding a comment to that effect.